lib/commands/web.js

```diff
@@ -0,0 +1,189 @@
+import { retryInterval } from 'asyncbox';
+import _ from 'lodash';
+import { errors } from 'mobile-json-wire-protocol';
+import logger from '../logger';
+
+
+let commands = {}, helpers = {}, extensions = {};
+
+const ELEMENT_OFFSET = 5000;
+
+commands.setUrl = async function (url) {
+  logger.debug(`Attempting to set url '${url}'`);
+  if (!this.isWebContext()) {
+    // in the future, detect whether we have a UIWebView that we can use to
+    // make sense of this command. For now, and otherwise, it's a no-op
+    throw new errors.NotImplementedError();
+  } else {
```

Comment:
and following (that's what i had in mind originally, and the benefit---one less level of indentation! one less pair of braces!)